src/murmur/Messages.cpp

```diff
@@ -1807,17 +1807,34 @@ void Server::msgContextAction(ServerUser *uSource, MumbleProto::ContextAction &m
 	emit contextAction(uSource, u8(msg.action()), session, id);
 }
 
+/// @param str The std::string to convert
+/// @param maxSize The maximum allowed size for this string
```

Comment:
What is the size specified in? Bytes? Characters? Code points?

Reply:
It's the size of the string. As in `str.size()` I guess in the end that means bytes.